Timer/js/index.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * Clear object value
+ * @param {object} element
+ */
+function clear(element) {
+  element.value = '';
+}
+
+/**
+ * Disable object
+ * @param {object} element
+ */
+function disable(element) {
+  element.setAttribute('disabled', 'disabled');
+}
+
+/**
+ * Enable object
+ * @param {object} element
+ */
+function enabled(element) {
+  element.removeAttribute('disabled');
+}
+
+/**
+ * Add zeros to result
+ */
+function addZero(element) {
+  element.value = (`0${element.value}`).slice(-2);
+}
+
+/**
+ * Loop function from parameter
+ * @param {string} func
+ */
+function loop(func) {
+  for (let key in getValues) {
+
+    if (getValues.hasOwnProperty(key)) {
+      func(getValues[key]);
+    }
+
+  }
+}
+
+/**
+ * Get seconds count
+ * @returns {number}
+ */
+function getTime() {
+  let time = 0;
+  let toSeconds = 3600;
+
+  for (let key in getValues) {
+
+    if (getValues.hasOwnProperty(key)) {
+      console.log(getValues[key].value);
+      time += (parseInt(getValues[key].value) * toSeconds);
+      toSeconds /= 60;
+
+    }
+
+  }
+  return time;
+}
+
+/**
+ * Stop timer
+ */
+function stop() {
+
+  alert(message.value);
+
+  loop(enabled);
+  clear(message);
+  message.style.display = 'inline-block';
```

Comment:
Manipulate, please, classes and ID but not CSS styles in the JS. Create class with display: inline-block property and add this class, if you need inline-block type of div